### ISSUE TITLE
Bug 796733 - warning: multiple use of section label 'xxx' while adding section 

### ISSUE THREAD

**Issue by Kochise** [doxygen/doxygen#6605]
Get this error because I add this to each file header :

```
///
/// \file	xxx.yyy
...
/// \section	LICENSE
///
/// Start of the Copyright Notice \n
/// zzz
/// End of the Copyright Notice \n
/// \n
///

```
No problem with 1.8.6

**Comment by Kochise** [comment]
The '...' part is additional doxygen informations. The problem doesn't show up until you deal with several files with the same file header featuring the same '__section__' part.

**Comment by albert-github** [comment]
When I understand it well you need to have the license text in multiple files.

Doxygen needs the `section` labels to be unique so that is why you will get the warning message.
The question for me is as well why use a section? as this would mean that in table of contents this would appear as well. When it is necessary to use a section I would think about an ALIAS with an argument so that when including the license text one can make it unique.

The solution I would use is:
- create a file  in a separate directory (and add this directory to the `EXAMPLE_PATH`) with a contents like:
    `<h1> LICENSE</h1>`
    ` `
    `Start of the Copyright Notice <br>`
    `zzz<br>`
    `End of the Copyright Notice <br>`
    `<br>`
- use `\includedoc` to include it in the relevant files

another solution would be similar but with snippets and `\snippetdoc`.


Note: I cannot get all the problems here as I'm missing  a, small, self contained example (source+config file) that shows the problem?

**Comment by Kochise** [comment]
Will try to provide it, but as I said, it works in 1.8.6, so why is there regression on this ?

**Comment by albert-github** [comment]
Would be good to have the example so we can see what the deeper cause is (in his case)
Sometimes some side effects, unexpected, happen when there are changes (and between 1.8.6 and now there have been quite a few changes / improvements).
Another possibility might be that in 1.8.6 the problem was present as well but, incorrectly, no message was given.

**Comment by Kochise** [comment]
Sounds fair, will try to provide a global example of my issues in a separate ZIP file a bit later.

**Comment by albert-github** [comment]
@Kochise Any possibility of getting the example or has the problem been resolved?

**Comment by albert-github** [comment]
Unfortunately no further response, (might be fixed in the mean time by e.g. https://github.com/doxygen/doxygen/issues/6779 or user found its problem and has fixed it) setting to incomplete problem report.

**Comment by Kochise** [comment]
Forgot about this one, will try to check if recent version solves the issue.

Thank for the reminder.

**Comment by Kochise** [comment]
# tl;dr

Tested : warning still present in **1.9.6**

```batch
C:/Users/Kochise/Downloads/doxygen-1.9.6.windows.x64.bin/test_2.c:3: warning: multiple use of section label 'LICENSE' while adding section, (first occurrence: C:/Users/Kochise/Downloads/doxygen-1.9.6.windows.x64.bin/test_1.c, line 3)
```

[test_1.c.txt](https://github.com/doxygen/doxygen/files/10955120/test_1.c.txt)
[test_2.c.txt](https://github.com/doxygen/doxygen/files/10955122/test_2.c.txt)

# Complete test procedure

1. From https://github.com/doxygen/doxygen/releases/tag/Release_1_9_6
1. Download `doxygen-1.9.6.windows.x64.bin.zip`
1. Extract it inside `doxygen-1.9.6.windows.x64.bin` folder
1. Add the two `test_x.c` files inside, alongside with `doxygen.exe`
1. Create `doxygen` subfolder, go inside it
1. From https://github.com/Kochise/batch-doxygen_generate
1. Download files into the `doxygen` subfolder you created above
1. Execute `doxygen_template_generate.bat`
1. Edit generated `doxygen-1.9.6.windows.x64.bin.doxygen` if needed
1. Execute `doxygen_documentation_generate.bat`
1. **Doxygen** should be run, the `.html` file opened at the end
1. Console output is as follow :

<details>
<summary>Console output</summary>
<p>

```batch
C:\Users\Kochise\Downloads\doxygen-1.9.6.windows.x64.bin\doxygen>doxygen_documentation_generate.bat
vpath=doxygen-1.9.6.windows.x64.bin
Fichier introuvable
Doxygen version used: 1.9.6
Searching for include files...
Searching for example files...
Searching for images...
Searching for dot files...
Searching for msc files...
Searching for dia files...
Searching for files to exclude
Searching INPUT for files to process...
Searching for files in directory C:/Users/Kochise/Downloads/doxygen-1.9.6.windows.x64.bin
Searching for files in directory C:/Users/Kochise/Downloads/doxygen-1.9.6.windows.x64.bin/doxygen
Reading and parsing tag files
Parsing files
Reading C:/Users/Kochise/Downloads/doxygen-1.9.6.windows.x64.bin/doxygen/README.md...
Preprocessing C:/Users/Kochise/Downloads/doxygen-1.9.6.windows.x64.bin/test_1.c...
Parsing file C:/Users/Kochise/Downloads/doxygen-1.9.6.windows.x64.bin/test_1.c...
Preprocessing C:/Users/Kochise/Downloads/doxygen-1.9.6.windows.x64.bin/test_2.c...
Parsing file C:/Users/Kochise/Downloads/doxygen-1.9.6.windows.x64.bin/test_2.c...
C:/Users/Kochise/Downloads/doxygen-1.9.6.windows.x64.bin/test_2.c:3: warning: multiple use of section label 'LICENSE' while adding section, (first occurrence: C:/Users/Kochise/Downloads/doxygen-1.9.6.windows.x64.bin/test_1.c, line 3)
Building macro definition list...
Building group list...
Building directory list...
Building namespace list...
Building file list...
Building class list...
Building concept list...
Computing nesting relations for classes...
Associating documentation with classes...
Associating documentation with concepts...
Building example list...
Searching for enumerations...
Searching for documented typedefs...
Searching for members imported via using declarations...
Searching for included using directives...
Searching for documented variables...
Building interface member list...
Building member list...
Searching for friends...
Searching for documented defines...
Computing class inheritance relations...
Computing class usage relations...
Flushing cached template relations that have become invalid...
Computing class relations...
Add enum values to enums...
Searching for member function documentation...
Creating members for template instances...
Building page list...
Search for main page...
Computing page relations...
Determining the scope of groups...
Sorting lists...
Determining which enums are documented
Computing member relations...
Building full member lists recursively...
Adding members to member groups.
Computing member references...
Inheriting documentation...
Generating disk names...
Adding source references...
Adding xrefitems...
Sorting member lists...
Setting anonymous enum type...
Computing dependencies between directories...
Generating citations page...
Counting members...
Counting data structures...
Resolving user defined references...
Finding anchors and sections in the documentation...
Transferring function references...
Combining using relations...
Adding members to index pages...
Correcting members for VHDL...
Computing tooltip texts...
Generating style sheet...
Generating search indices...
Generating example documentation...
Generating file sources...
Parsing code for file doxygen/README.md...
Parsing code for file test_1.c...
Parsing code for file test_2.c...
Generating file documentation...
Generating docs for file doxygen/README.md...
Generating docs for file test_1.c...
Generating docs for file test_2.c...
Generating page documentation...
Generating docs for page md_doxygen_README...
C:/Users/Kochise/Downloads/doxygen-1.9.6.windows.x64.bin/doxygen/README.md:9: warning: Found unknown command '\Program'
C:/Users/Kochise/Downloads/doxygen-1.9.6.windows.x64.bin/doxygen/README.md:9: warning: Found unknown command '\Graphvizyourinstalledversionbecausehavingitinthefoldernameisobviouslyveryimportant'
C:/Users/Kochise/Downloads/doxygen-1.9.6.windows.x64.bin/doxygen/README.md:9: warning: Found unknown command '\bin'
Generating group documentation...
Generating class documentation...
Generating concept documentation...
Generating namespace index...
Generating graph info page...
Generating directory documentation...
Generating index page...
Generating page index...
Generating module index...
Generating namespace index...
Generating namespace member index...
Generating concept index...
Generating annotated compound index...
Generating alphabetical compound index...
Generating hierarchical class index...
Generating graphical class hierarchy...
Generating member index...
Generating file index...
Generating file member index...
Generating example index...
finalizing index lists...
writing tag file...
Running plantuml with JAVA...
Running dot...
Running dot for graph 1/1
C:/Users/Kochise/Downloads/doxygen-1.9.6.windows.x64.bin/doxygen/html/graph_legend.dot:1: error: Problems running dot: exit code=-1, command='dot.exe', arguments='"C:/Users/Kochise/Downloads/doxygen-1.9.6.windows.x64.bin/doxygen/html/graph_legend.dot" -Tpng -o "C:/Users/Kochise/Downloads/doxygen-1.9.6.windows.x64.bin/doxygen/html/graph_legend.png"'

type lookup cache used 0/65536 hits=0 misses=0
symbol lookup cache used 0/65536 hits=0 misses=0
finished...

C:\Users\Kochise\Downloads\doxygen-1.9.6.windows.x64.bin\doxygen>
```

</p>
</details>

Yet expected output is output as expected :

![doxygen-1 9 6 windows x64 bin](https://user-images.githubusercontent.com/23077417/224649276-4f6ca9e9-0bc3-4c14-ba7c-f283052bf417.png)

**Comment by albert-github** [comment]
1. Please don't add  references to external sources as they might not be persistent.
2. you request to run a certain `bat` file, but this runs only on Windows so other people with Linux cannot verify it. Please add the result of the procedure to the issue.

So all in all a self contained example (source+configuration file in a, compressed, tar or zip file) is needed that allows us to reproduce the problem? Please don't add external links as they might not be persistent.

I looked at the log file and the relevant output is:
```
Preprocessing C:/Users/Kochise/Downloads/doxygen-1.9.6.windows.x64.bin/test_1.c...
Parsing file C:/Users/Kochise/Downloads/doxygen-1.9.6.windows.x64.bin/test_1.c...
Preprocessing C:/Users/Kochise/Downloads/doxygen-1.9.6.windows.x64.bin/test_2.c...
Parsing file C:/Users/Kochise/Downloads/doxygen-1.9.6.windows.x64.bin/test_2.c...
C:/Users/Kochise/Downloads/doxygen-1.9.6.windows.x64.bin/test_2.c:3: warning: multiple use of section label 'LICENSE' while adding section, (first occurrence: C:/Users/Kochise/Downloads/doxygen-1.9.6.windows.x64.bin/test_1.c, line 3)
```
In **both** the test_1.c and test_2.c I see:
```
/// \section    LICENSE
```
in doxygen labels / anchors should be unique over the complete source, but here we have twice the label / anchor `LICENSE` and this is not allowed in doxygen.

  (Note: the "problem" is already reproducible with just the 2 files and an empty `Doxyfile`).

**Comment by albert-github** [comment]
A possible solution for you might be to replace:
```
/// \section    LICENSE
```
with
```
/// # LICENSE
```

**Comment by Kochise** [comment]
Ok, thank for the feedback, I'll see what's file-wise and what's project-wide.

Regards.
